R/utils.R

```diff
@@ -1,11 +1,20 @@
 #'@importFrom httr stop_for_status GET user_agent content status_code
 pv_query <- function(params, reformat = TRUE, ...){
+  if(length(params) > 1){
+    data <- lapply(as.list(params), pv_query, reformat = reformat)
+    if(reformat == TRUE){
+      return(do.call(rbind, data))
+    } else {
+      return(data)
```

Comment:
Fair! Hmn. I mean, noodling on it if we can assume reformatting by default, I'm fine with it.